lib/coinbase/wallet.rb

```diff
@@ -367,6 +367,14 @@ def claimable_balance(asset_id, mode: :default, options: {})
       default_address.claimable_balance(asset_id, mode: mode, options: options)
     end
 
+    # Enumerates the historical balances for a given asset belonging of default address.
```

Comment:
Enumerates the historical balances for a given asset belonging to the default address of the wallet